### ISSUE TITLE
[BUG] bash_functions.sh causes issues on a Github Runner because it runs in interactive mode.

### ISSUE THREAD

**Issue by middle-membership600** [OpenCHAMI/deployment-recipes#65]
**Describe the bug**
When running the quickstart on a Github Runner, I get errors on step 5:
```
source bash_functions.sh
# Download the root ca so you can validate the ssl certificates included with your system
get_ca_cert > cacert.pem
```
This returns an error because the bash_functions.sh expect to be run interactively–because of the -it flag–(a Github runner is not interactive).

**Screenshots**
![Screenshot 2024-07-09 at 9 45 14 AM](https://github.com/OpenCHAMI/deployment-recipes/assets/105831328/ff8d3b8a-9917-465b-bc6d-4ef6eadd4d5a)

**Comment by synackd** [comment]
@middle-membership600 I edited your comment to use a code block so it's more clear which commands you're running.

**Comment by synackd** [comment]
Does this branch work for you?
https://github.com/synackd/deployment-recipes/tree/quickstart-bash-functions-no-it

**Comment by middle-membership600** [comment]
I have just tested this and this new branch works.

**Comment by synackd** [comment]
Solved with the merge of #69.
